src/main/java/net/glowstone/net/message/play/game/TitleMessage.java

```diff
@@ -41,9 +41,12 @@ public TitleMessage(Action action) {
     }
 
     public static TitleMessage[] fromTitle(Title title) {
+        TextMessage titleMessage = title.getTitle() != null ? asTextMessage(BaseComponent.toPlainText(title.getTitle())) : asTextMessage(null);
```

Comment:
Running the asTextMessage method will create a new TextMessage, but with an empty body. This makes the error in #392 go away, because before doing BaseComponent.toPlainText was throwing the error because title.getTitle was null, or title.getSubTitle was null.